### ISSUE TITLE
Time Series Tab missing latest run

### ISSUE THREAD

**Issue by lukasfolle** [tensorflow/tensorboard#5312]
Scalar Tab shows all runs, but the Time Series tab is missing always the latest run. To me it seems like this is a bug of the gui as I can't see the lower part of the scroll bar next to the runs in the Time Series tab.

### Diagnostics

<details>
<summary>Diagnostics output</summary>

``````
--- check: autoidentify
INFO: diagnose_tensorboard.py version e43767ef2b648d0d5d57c00f38ccbd38390e38da

--- check: general
INFO: sys.version_info: sys.version_info(major=3, minor=8, micro=6, releaselevel='final', serial=0)
INFO: os.name: nt
INFO: os.uname(): N/A
INFO: sys.getwindowsversion(): sys.getwindowsversion(major=10, minor=0, build=19042, platform=2, service_pack='')

--- check: package_management
INFO: has conda-meta: True
INFO: $VIRTUAL_ENV: None

--- check: installed_packages
WARNING: no installation among: ['tb-nightly', 'tensorboard', 'tensorflow-tensorboard']
WARNING: no installation among: ['tensorflow', 'tensorflow-gpu', 'tf-nightly', 'tf-nightly-2.0-preview', 'tf-nightly-gpu', 'tf-nightly-gpu-2.0-preview']
WARNING: no installation among: ['tensorflow-estimator', 'tensorflow-estimator-2.0-preview', 'tf-estimator-nightly']

--- check: tensorboard_python_version
INFO: tensorboard.version.VERSION: '2.5.0'

--- check: tensorflow_python_version
Traceback (most recent call last):
  File "C:/Users/follels/Documents/arthritis/deep_learning/tb_debug.py", line 522, in main
    suggestions.extend(check())
  File "C:/Users/follels/Documents/arthritis/deep_learning/tb_debug.py", line 75, in wrapper
    result = fn()
  File "C:/Users/follels/Documents/arthritis/deep_learning/tb_debug.py", line 278, in tensorflow_python_version
    import tensorflow as tf
ModuleNotFoundError: No module named 'tensorflow'

--- check: tensorboard_data_server_version
INFO: no data server installed

--- check: tensorboard_binary_path
INFO: which tensorboard: b'C:\\Users\\follels\\.conda\\envs\\arthritis-gpu\\Scripts\\tensorboard.exe\r\n'

--- check: addrinfos
socket.has_ipv6 = True
socket.AF_UNSPEC = <AddressFamily.AF_UNSPEC: 0>
socket.SOCK_STREAM = <SocketKind.SOCK_STREAM: 1>
socket.AI_ADDRCONFIG = <AddressInfo.AI_ADDRCONFIG: 1024>
socket.AI_PASSIVE = <AddressInfo.AI_PASSIVE: 1>
Loopback flags: <AddressInfo.AI_ADDRCONFIG: 1024>
Loopback infos: [(<AddressFamily.AF_INET6: 23>, <SocketKind.SOCK_STREAM: 1>, 0, '', ('::1', 0, 0, 0)), (<AddressFamily.AF_INET: 2>, <SocketKind.SOCK_STREAM: 1>, 0, '', ('127.0.0.1', 0))]
Wildcard flags: <AddressInfo.AI_PASSIVE: 1>
Wildcard infos: [(<AddressFamily.AF_INET6: 23>, <SocketKind.SOCK_STREAM: 1>, 0, '', ('::', 0, 0, 0)), (<AddressFamily.AF_INET: 2>, <SocketKind.SOCK_STREAM: 1>, 0, '', ('0.0.0.0', 0))]

--- check: readable_fqdn
INFO: socket.getfqdn(): 'm3-214887.medads.uk-erlangen.de'

--- check: stat_tensorboardinfo
INFO: directory: C:\Users\follels\AppData\Local\Temp\.tensorboard-info
INFO: os.stat(...): os.stat_result(st_mode=16895, st_ino=1407374883694723, st_dev=781669087, st_nlink=1, st_uid=0, st_gid=0, st_size=4096, st_atime=1631640122, st_mtime=1631611364, st_ctime=1612976234)
INFO: mode: 0o40777

--- check: source_trees_without_genfiles
INFO: tensorboard_roots (1): ['C:\\Users\\follels\\.conda\\envs\\arthritis-gpu\\lib\\site-packages']; bad_roots (0): []

--- check: full_pip_freeze
INFO: pip freeze --all:
absl-py @ file:///D:/bld/absl-py_1606235688386/work
aiohttp @ file:///D:/bld/aiohttp_1610358740361/work
aiohttp-cors==0.7.0
aioredis==1.3.1
argon2-cffi @ file:///D:/bld/argon2-cffi_1605217273648/work
async-generator==1.10
async-timeout==3.0.1
attrs @ file:///home/conda/feedstock_root/build_artifacts/attrs_1605083924122/work
ax-platform==0.1.19
backcall @ file:///home/conda/feedstock_root/build_artifacts/backcall_1592338393461/work
backports.functools-lru-cache==1.6.1
bleach @ file:///home/conda/feedstock_root/build_artifacts/bleach_1600454382015/work
blessings==1.7
blinker==1.4
botorch==0.3.3
brotlipy==0.7.0
cached-property==1.5.1
cachetools @ file:///home/conda/feedstock_root/build_artifacts/cachetools_1593420445823/work
captum==0.3.0
certifi==2021.5.30
cffi @ file:///D:/bld/cffi_1606601395031/work
chardet @ file:///D:/bld/chardet_1602255508636/work
click==7.1.2
cloudpickle @ file:///tmp/build/80754af9/cloudpickle_1598884132938/work
colorama @ file:///home/conda/feedstock_root/build_artifacts/colorama_1602866480661/work
colorful==0.5.4
cryptography @ file:///D:/bld/cryptography_1610338909973/work
curses @ file:///C:/Users/follels/Downloads/curses-2.2.1%2Butf8-cp38-cp38-win_amd64.whl
cycler==0.10.0
cytoolz==0.11.0
dask @ file:///tmp/build/80754af9/dask-core_1607706933335/work
decorator==4.4.2
defusedxml==0.6.0
Deprecated @ file:///home/conda/feedstock_root/build_artifacts/deprecated_1589409885623/work
efficientnet-pytorch==0.6.3
entrypoints @ file:///home/conda/feedstock_root/build_artifacts/entrypoints_1605121927639/work/dist/entrypoints-0.3-py2.py3-none-any.whl
et-xmlfile==1.0.1
filelock==3.0.12
fsspec @ file:///home/conda/feedstock_root/build_artifacts/fsspec_1608050797851/work
future @ file:///D:/bld/future_1610147454374/work
fvcore==0.1.5.post20210722
google-api-core==1.26.0
google-auth @ file:///home/conda/feedstock_root/build_artifacts/google-auth_1608136875028/work
google-auth-oauthlib==0.4.1
googleapis-common-protos==1.52.0
gpustat==0.6.0
gpytorch==1.3.1
grpcio @ file:///D:/bld/grpcio_1610146642916/work
h5py @ file:///D:/bld/h5py_1604753864980/work
hiredis==1.1.0
idna @ file:///home/conda/feedstock_root/build_artifacts/idna_1593328102638/work
imageio @ file:///tmp/build/80754af9/imageio_1594161405741/work
importlib-metadata @ file:///D:/bld/importlib-metadata_1610355403747/work
iopath==0.1.9
ipydatawidgets @ file:///home/conda/feedstock_root/build_artifacts/ipydatawidgets_1609929402090/work
ipykernel @ file:///D:/bld/ipykernel_1607737448354/work/dist/ipykernel-5.4.2-py3-none-any.whl
ipython @ file:///D:/bld/ipython_1604159844946/work
ipython-genutils==0.2.0
ipywidgets @ file:///home/conda/feedstock_root/build_artifacts/ipywidgets_1609995587151/work
jdcal==1.4.1
jedi @ file:///D:/bld/jedi_1610146948662/work
Jinja2==2.11.2
joblib @ file:///home/conda/feedstock_root/build_artifacts/joblib_1607956439537/work
jsonschema @ file:///home/conda/feedstock_root/build_artifacts/jsonschema_1602551949684/work
jupyter-client @ file:///home/conda/feedstock_root/build_artifacts/jupyter_client_1610375432619/work
jupyter-core @ file:///D:/bld/jupyter_core_1605735280365/work
jupyterlab-pygments @ file:///home/conda/feedstock_root/build_artifacts/jupyterlab_pygments_1601375948261/work
jupyterlab-widgets @ file:///home/conda/feedstock_root/build_artifacts/jupyterlab_widgets_1609173350931/work
K3D @ file:///home/conda/feedstock_root/build_artifacts/k3d_1605537932493/work
kiwisolver @ file:///D:/bld/kiwisolver_1610099948875/work
Markdown @ file:///home/conda/feedstock_root/build_artifacts/markdown_1603668500820/work
MarkupSafe @ file:///D:/bld/markupsafe_1610127683324/work
matplotlib @ file:///D:/bld/matplotlib-suite_1605180506412/work
mistune @ file:///D:/bld/mistune_1610113104461/work
mkl-service==2.3.0
monai==0.6.0
msgpack==1.0.2
multidict @ file:///D:/bld/multidict_1610319170773/work
munch==2.5.0
nbclient @ file:///home/conda/feedstock_root/build_artifacts/nbclient_1602859080374/work
nbconvert @ file:///D:/bld/nbconvert_1605402041690/work
nbformat @ file:///home/conda/feedstock_root/build_artifacts/nbformat_1602732862338/work
nest-asyncio @ file:///home/conda/feedstock_root/build_artifacts/nest-asyncio_1605195931949/work
networkx @ file:///tmp/build/80754af9/networkx_1598376031484/work
nibabel @ file:///home/conda/feedstock_root/build_artifacts/nibabel_1606620997676/work
notebook @ file:///D:/bld/notebook_1610575486218/work
numpy @ file:///D:/bld/numpy_1610324782150/work
nvidia-ml-py3==7.352.0
oauthlib==3.0.1
olefile @ file:///home/conda/feedstock_root/build_artifacts/olefile_1602866521163/work
opencensus==0.7.12
opencensus-context==0.1.2
openpyxl @ file:///tmp/build/80754af9/openpyxl_1598113097404/work
packaging @ file:///home/conda/feedstock_root/build_artifacts/packaging_1607785313469/work
pandas @ file:///D:/bld/pandas_1609079442959/work
pandocfilters==1.4.2
parso @ file:///home/conda/feedstock_root/build_artifacts/parso_1607618318316/work
patsy==0.5.1
pickleshare @ file:///home/conda/feedstock_root/build_artifacts/pickleshare_1602536217715/work
Pillow @ file:///C:/ci/pillow_1609786840597/work
pip==20.3.3
plotly==4.14.3
portalocker @ file:///D:/bld/portalocker_1610360408397/work
pretrainedmodels==0.7.4
prometheus-client @ file:///home/conda/feedstock_root/build_artifacts/prometheus_client_1605543085815/work
prompt-toolkit @ file:///home/conda/feedstock_root/build_artifacts/prompt-toolkit_1610127285153/work
protobuf==3.14.0
psutil==5.7.2
py-spy==0.3.4
pyasn1==0.4.8
pyasn1-modules==0.2.7
pycparser @ file:///home/conda/feedstock_root/build_artifacts/pycparser_1593275161868/work
pydicom @ file:///home/conda/feedstock_root/build_artifacts/pydicom_1607562682046/work
pyelastix==1.2
Pygments @ file:///home/conda/feedstock_root/build_artifacts/pygments_1607352093725/work
PyJWT @ file:///home/conda/feedstock_root/build_artifacts/pyjwt_1609976754451/work
pyOpenSSL @ file:///home/conda/feedstock_root/build_artifacts/pyopenssl_1608055815057/work
pyparsing==2.4.7
pyreadline @ file:///D:/bld/pyreadline_1602253139225/work
pyrsistent @ file:///D:/bld/pyrsistent_1610147004705/work
PySocks @ file:///D:/bld/pysocks_1610291742246/work
python-dateutil==2.8.1
pytorch-lightning @ file:///home/conda/feedstock_root/build_artifacts/pytorch-lightning_1610492789276/work
pytz @ file:///home/conda/feedstock_root/build_artifacts/pytz_1608904108784/work
PyWavelets @ file:///C:/ci/pywavelets_1601658407916/work
pywin32==228
pywinpty @ file:///D:/bld/pywinpty_1602377663724/work
PyYAML==5.3.1
pyzmq @ file:///D:/bld/pyzmq_1610626967848/work
ray==1.2.0
redis==3.5.3
requests @ file:///home/conda/feedstock_root/build_artifacts/requests_1608156231189/work
requests-oauthlib @ file:///home/conda/feedstock_root/build_artifacts/requests-oauthlib_1595492159598/work
retrying==1.3.3
rsa @ file:///home/conda/feedstock_root/build_artifacts/rsa_1610280765105/work
scikit-image==0.17.2
scikit-learn @ file:///D:/bld/scikit-learn_1608656402924/work
scipy @ file:///C:/ci/scipy_1604596260408/work
seaborn @ file:///tmp/build/80754af9/seaborn_1608578541026/work
segmentation-models-pytorch==0.1.3
Send2Trash==1.5.0
setuptools==49.6.0.post20210108
SimpleITK==2.0.2
sip==4.19.13
six @ file:///home/conda/feedstock_root/build_artifacts/six_1590081179328/work
statsmodels @ file:///D:/bld/statsmodels_1612273760174/work
tabulate @ file:///home/conda/feedstock_root/build_artifacts/tabulate_1614001031686/work
tensorboard @ file:///home/builder/ktietz/aggregate/tensorflow_recipes/ci_baze37/tensorboard_1622025863084/work/tensorboard-2.5.0-py3-none-any.whl
tensorboard-plugin-wit @ file:///home/conda/feedstock_root/build_artifacts/tensorboard-plugin-wit_1603349307347/work/tensorboard_plugin_wit-1.7.0-py3-none-any.whl
tensorboardX==2.1
termcolor==1.1.0
terminado @ file:///D:/bld/terminado_1609796834027/work
testpath==0.4.4
threadpoolctl @ file:///tmp/tmp79xdzxkt/threadpoolctl-2.1.0-py3-none-any.whl
tifffile==2020.10.1
timm==0.3.2
tir @ file:///C:/Users/follels/Documents/tir
toolz @ file:///tmp/build/80754af9/toolz_1601054250827/work
torch==1.8.2
torchaudio==0.8.2
torchsampler @ https://github.com/ufoym/imbalanced-dataset-sampler/archive/master.zip
torchvision==0.9.2
tornado @ file:///D:/bld/tornado_1610094858390/work
tqdm @ file:///home/conda/feedstock_root/build_artifacts/tqdm_1610419111160/work
traitlets @ file:///home/conda/feedstock_root/build_artifacts/traitlets_1602771532708/work
traittypes @ file:///home/conda/feedstock_root/build_artifacts/traittypes_1600843364635/work
typeguard==2.11.1
typing-extensions @ file:///home/conda/feedstock_root/build_artifacts/typing_extensions_1602702424206/work
urllib3 @ file:///home/conda/feedstock_root/build_artifacts/urllib3_1605225765842/work
wcwidth @ file:///home/conda/feedstock_root/build_artifacts/wcwidth_1600965781394/work
webencodings==0.5.1
Werkzeug==1.0.1
wheel==0.36.2
widgetsnbextension @ file:///D:/bld/widgetsnbextension_1605475695976/work
win-inet-pton @ file:///D:/bld/win_inet_pton_1610147566231/work
wincertstore==0.2
wrapt @ file:///D:/bld/wrapt_1610095100080/work
xlrd @ file:///home/conda/feedstock_root/build_artifacts/xlrd_1610224409810/work
yacs==0.1.6
yarl @ file:///D:/bld/yarl_1610354399983/work
zipp @ file:///home/conda/feedstock_root/build_artifacts/zipp_1603668650351/work

``````

</details>

![Scalars_TB](https://user-images.githubusercontent.com/26490449/133428706-c3992b69-8fb3-471b-9b6a-625e9016670b.png)
![TimeSeries_TB](https://user-images.githubusercontent.com/26490449/133428713-23d1e155-d78e-4ce5-bc28-6c7db5d30c05.png)

**Comment by bileschi** [comment]
Thanks for filing the issue and including the diagnostics!

From the diagnostics, it appears this behavior was recorded using TensorBoard at `2.5.0`

   I was able to reproduce this behavior locally in `2.5.0` but not in the most recent release, `2.6.0`.  Is it possible to upgrade your TensorBoard install?

**Comment by lukasfolle** [comment]
Thank you for your help, version 2.6 fixes this issue.
Though I am not entirely sure if it was fixed in 2.6 or simply disappeared in 2.6 :D
